roles/ceph-fetch-keys/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: find ceph keys
+  shell: ls -1 /etc/ceph/*.keyring
+  changed_when: false
+  register: ceph_keys
+
+- name: set keys permissions
+  file:
+    path: "{{ item }}"
+    mode: 0600
+    owner: root
```

Comment:
@leseb yeah, maybe so. Do we need to fix that here as well? https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/tasks/ceph_keys.yml#L28

Running a ``ceph osd tree`` on the new OSD does give me an error about how it can't find the keyrings. Maybe this is indicating a permissions problem? They look ok to me though.

```
vagrant@vagrant:~$ sudo ceph osd tree
2016-02-18 16:37:02.236868 7f27b5194700 -1 auth: unable to find a keyring on /etc/ceph/ceph.client.admin.keyring,/etc/ceph/ceph.keyring,/etc/ceph/keyring,/etc/ceph/keyring.bin: (2) No such file or directory
2016-02-18 16:37:02.237400 7f27b5194700 -1 monclient(hunting): ERROR: missing keyring, cannot use cephx for authentication
2016-02-18 16:37:02.237960 7f27b5194700  0 librados: client.admin initialization error (2) No such file or directory
Error connecting to cluster: ObjectNotFound
vagrant@vagrant:~$ sudo ls -l /var/lib/ceph/bootstrap-osd
total 4
-rw------- 1 ceph ceph 71 Feb 17 23:18 ceph.keyring
```